llvm/cmake/modules/HandleLLVMOptions.cmake

```diff
@@ -1272,6 +1272,11 @@ if(LLVM_ENABLE_EXPERIMENTAL_DEPSCAN)
     append("-fdepscan-prefix-map=${source_root}=/^source" CMAKE_C_FLAGS CMAKE_CXX_FLAGS)
     append("-fdepscan-prefix-map=${CMAKE_BINARY_DIR}=/^build" CMAKE_C_FLAGS CMAKE_CXX_FLAGS)
 
+    # Create a LLDB init file to remap prefix map back to original path.
+    # TODO: Currently, we only remap source and build directory.
```

Comment:
This isn't actually saying what's left TODO, just what's done. I suggest mentioning the SDK and toolchain.